### PR TITLE
Add `createAndRegister` (PHNX-14939)

### PR DESCRIPTION
**Pull request by JonathanAtCenterEdge**
# Motivations
We need `createAndRegister` for account linkage in Webstores

# Modifications
- Add `POST createAndRegister.json` to our spec
- Condense our enrollment tags to just `Enrollment`
- Remove an added `.yaml` on the account query response, surprisingly this didn't affect our spec but just made our spec definition ugly

https://centeredge.atlassian.net/browse/PHNX-14939

### CODE REVIEW

File: openapi/components/responses/createAndRegisterResponse.yaml
Comment (brantburnett) [inline openapi/components/responses/createAndRegisterResponse.yaml:1]: This file should technically be in the `schemas` directory, representing a schema for a response body. The `responses` directory is for entire `response` objects, meaning the response information included beneath a particular status code such as found in `unauthorized.yaml`.

This was done incorrectly previously and I've been gradually fixing the invalid cases.

**Review by brantburnett — CHANGES_REQUESTED** [review]

File: openapi/components/responses/createAndRegisterResponse.yaml
Comment (JonathanAtCenterEdge) [reply openapi/components/responses/createAndRegisterResponse.yaml:1]: So, `createAndRegisterCardCreatedSuccess.yaml` should be in `/responses`, and `createAndRegisterResponse.yaml` should be in `/schemas`?

Comment (brantburnett) [reply openapi/components/responses/createAndRegisterResponse.yaml:1]: If you feel that it's worth creating `createAndRegisterCardCreatedSuccess.yaml`, it's also valid for simple, single use cases to leave that where it is, as an embedded definition within the path/operation definition in the `paths` directory.

**Review by brantburnett — APPROVED** [review]

**Review by RobRobertsCE — APPROVED** [review]

**Pull request merged**